src/odemis/driver/orsay.py

```diff
@@ -1389,3 +1440,1247 @@ def terminate(self):
+        self._flybackTimeConnector = OrsayParameterConnector(self.flybackTime, self._ionColumn.FlybackTime)
+        self._blankingDelayConnector = OrsayParameterConnector(self.blankingDelay, self._ionColumn.BlankingDelay)
+        self._rotationConnector = OrsayParameterConnector(self.rotation, self._ionColumn.ObjectiveScanAngle)
+        self._dwellTimeConnector = OrsayParameterConnector(self.dwellTime, self._ionColumn.PixelTime,
+                                                           minpar=self._ionColumn.PixelTime_Minvalue,
+                                                           maxpar=self._ionColumn.PixelTime_Maxvalue)
+        self._contrastConnector = OrsayParameterConnector(self.contrast, self._sed.PMT, factor=0.01)
+        self._brightnessConnector = OrsayParameterConnector(self.brightness, self._sed.Level, factor=0.01)
+        self._operatingModeConnector = OrsayParameterConnector(self.operatingMode,
+                                                               self._datamodel.Scanner.OperatingMode,
+                                                               conversion={True: 1, False: 0})
+        # Subscribe to the parameter on the Orsay server
+        self._ionColumn.ImageSize.Subscribe(self._updateImageFormat)
+        self._ionColumn.ImageArea.Subscribe(self._updateTranslationResolution)
+
+        self._connectorList = [x for (x, _) in  # save only the names of the returned members
+                               inspect.getmembers(self,  # get all members of this FIB_source object
+                                                  lambda obj: type(obj) == OrsayParameterConnector  # get only the
+                                                  # OrsayParameterConnectors from all members of this FIB_source object
+                                                  )
+                               ]
+
+    def update_VAs(self):
+        """
+        Update the VA's. Should be called after reconnection to the server
+        """
+        self._updateImageFormat()
+        self._updateTranslationResolution()
+        for obj_name in self._connectorList:
+            getattr(self, obj_name).update_VA()
+
+    def _imageFormat_setter(self, value):
+        """
+        Setter of the imageFormat VA
+
+        :param (tuple (int, int)) value: The goal format of the image. Will be corrected to the closest available value
+            in IMAGEFORMAT_OPTIONS.
+        :return (tuple (int, int)): The actual image format set.
+        """
+        if value not in IMAGEFORMAT_OPTIONS:  # get the closest option available in IMAGEFORMAT_OPTIONS
+            value = min(IMAGEFORMAT_OPTIONS, key=lambda x: abs(x[0] - value[0]) + abs(x[1] - value[1]))
+        self.imageFormatUpdatedResolutionTranslation.clear()  # let it be known that image format is updating...
+        # ...resolution and translation
+
+        # get the old image format and determine the scale change
+        state = self._ionColumn.ImageSize.Actual
```

Comment:
Just look at the old value of the VA, no need to ask the server. As the VA hasn't been updated, you can just do `prev_value = self.imageFormat.value`